miniapps/nurbs/nurbs_ex1.cpp

```diff
@@ -3,6 +3,7 @@
 // Compile with: make nurbs_ex1
 //
 // Sample runs:  nurbs_ex1 -m square-nurbs.mesh -o 2 -no-ibp
+//               nurbs_ex1 -m square-nurbs.mesh -o 2 --weak-bc
```

Comment:
I'm not sure miniapps is the right place for these examples.  The miniapps are usually more complicated.  Where do folks think we should put these?

Reply:
I don't see a reason to move them at this point. Miniapps don't need to be complicated -- there are already a number of simple miniapps.

Reply:
Perhaps they can be moved to a subdirectory of examples. However, it seems the other subdirs seem to depend on compilation options (package used yes/no).